README.rst

```diff
@@ -5,7 +5,7 @@ Matrix Client SDK for Python
   :target: https://pypi.python.org/pypi/matrix-client/
   :alt: Latest Version
 
-This is a Matrix client-server SDK for Python 2.x.
+This is a Matrix client-server SDK for Python 3.x.
```

Comment:
Does this break Python 2.x compatibility? If not, I’d write `Python 2.x and 3.x`